src/snowflake/snowpark/_internal/proto/ast.proto

```diff
@@ -2853,4 +2862,4 @@ message UnaryOp {
     Neg neg = 1;
     Not not = 2;
   }
-}
+}
```

Comment:
newline

Reply:
This is codegened. I'll take care of this in a subsequent PR.